compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/utils/Utils.java

```diff
@@ -240,4 +247,74 @@ public static String getDatastore(CodeActionContext ctx) throws BalException {
             throw new BalException("error while reading persist configurations. " + e.getMessage());
         }
     }
+
+    public static String readStringValueFromAnnotation
+            (io.ballerina.compiler.syntax.tree.NodeList<AnnotationNode> annotationNodes, String annotation,
+             String field) {
+        if (annotationNodes == null) {
```

Comment:
`annotationNodes` are a list (or a collection). In an collection we usually check whether the collection is empty or not. Are there any case where this could be null ? If so, can't we check that from the caller method itself ?